politeiad/backendv2/tstorebe/plugins/comments/cmds.go

```diff
@@ -442,11 +442,8 @@ func (p *commentsPlugin) cmdNew(token []byte, payload string) (string, error) {
 	}
 
 	// Ensure no extra data provided if not allowed
-	if !p.allowExtraData && (n.ExtraData != "" || n.ExtraDataHint != "") {
-		return "", backend.PluginError{
-			PluginID:  comments.PluginID,
-			ErrorCode: uint32(comments.ErrorCodeExtraDataNotAllowed),
-		}
+	if err = p.validateExtraData(n.ExtraData, n.ExtraDataHint); err != nil {
```

Comment:
There's nothing wrong with this style.  It's commonly used in the go ecosystem.  This repo avoids it though because shorter lines are easier to read.  Same reason we have a 80 column width limit.

```
err = p.validateExtraData(n.ExtraData, n.ExtraDataHint)
if err != nil {
```